github_rules/github_repo_initial_access.yml

```diff
@@ -75,3 +75,24 @@ Tests:
         "actor": "cat",
         "user": ""
       }
+  -
+    Name: GitHub - Clone without Actor
+    ExpectedResult: true
+    Mocks:
+      - objectName: get_string_set
+        returnValue: >
+          "cat":"my-repo"
```

Comment:
I think we could remove the mocks on this test, just because we expect that the kv store shouldn't be called in this particular test case.  If it does get called, we would want this to error out.
 
Thanks for including a test for the code branch! 🙌🏻

Reply:
Good call - removed!